src/dk/aau/cs/ds306e18/tournament/rlbot/configuration/MatchConfig.java

```diff
@@ -0,0 +1,356 @@
+package dk.aau.cs.ds306e18.tournament.rlbot.configuration;
+
+import dk.aau.cs.ds306e18.tournament.rlbot.configuration.MatchConfigOptions.*;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Class from interfacing with a match config (usually called rlbot.cfg).
```

Comment:
```suggestion
 * Class for interfacing with a match config (usually called rlbot.cfg).
```

Just a typo, I imagine